lib/AlternativeNamesFieldArray/tests/AlternativeNamesFieldArray-test.js

```diff
@@ -0,0 +1,204 @@
+/* eslint-disable react/prop-types */
+
+import React from 'react';
+import { describe, beforeEach, it } from '@bigtest/mocha';
+import chai from 'chai';
+import spies from 'chai-spies';
+import { Form } from 'react-final-form';
+import arrayMutators from 'final-form-arrays';
+import { FieldArray } from 'react-final-form-arrays';
+
+import { Button } from '@folio/stripes/components';
+
+import { dummyMount, setupApplication } from '../../../tests/helpers';
+import AlternativeNamesFieldArray from '../AlternativeNamesFieldArray';
+
+import TasksFormInteractor from './interactor';
```

Comment:
What's all this about tasks in this file? Please be careful with copy-and-paste. 🙃

Reply:
Hm. Isn't it just a name? Again, laziness. Do you want me to change all the `task` names into `name` or `alternativeName` for Instance?

Reply:
It's incredibly frustrating to have to parse through new code when parts of it are irrelevant, since I have to hold in my head the parts that are relevant and discard the parts that aren't. I didn't even bother to look at these tests because it's just such an annoying and time-consuming thing to do.

And whenever someone works on those tests, they'll be repeating what you'd make me do right now if I was going to review it.

Reply:
you leave me perplexed.
i don't think anything is irrelevant. but i'm not good enough at bigtest, so i have to copy code and try to adapt. In my opinion I just didn't change the names, I can still do that